tasks/src/Zero-shot-classification/about.md

```diff
@@ -0,0 +1,9 @@
+## Use Cases
+
+### Text Classification
+
+Zero-shot text classification is a task in natural language processing (NLP) where a model is trained on a set of labeled examples but is then able to classify new examples from previously unseen classes by transferring knowledge from seen to unseen classes.
```

Comment:
I feel like this is a very good definition for summary part instead 😅 WDYT?

Reply:
ya, that would be great. I will do the necessary changes

Reply:
yes it's actually text classification https://huggingface.co/models?pipeline_tag=zero-shot-classification&sort=downloads